configuration/screensaver.md

```diff
@@ -1,28 +1,34 @@
 # Setting the screen saver/screen blanking
 
-## On the Console
+## On the console
 
-If you are using the Raspberry Pi solely on the console (no desktop GUI), you need to set the console blanking. The current setting, in seconds, can be displayed using
+By default, when running without a graphical desktop, Raspberry Pi OS will blank the screen after 10 minutes without user input: either mouse movement or keyboard input.
+
+The current setting, in seconds, can be displayed using:
 ```
 cat /sys/module/kernel/parameters/consoleblank
 ```
 
-Here, `consoleblank` is a kernel parameter. In order to be permanently set, it needs to be defined on the kernel command line.
+Here, `consoleblank` is a kernel parameter. In order to be permanently set, it needs to be defined on the kernel command line. To edit the kernel command line:
 
 ```
 sudo nano /boot/cmdline.txt
 ```
 
-Add `consoleblank=0` to turn screen blanking off completely, or edit it to set the number of seconds of inactivity before the console will blank. Note the kernel command line must be a single line of text.
+The file `/boot/cmdline.txt` contains a single line of text. Add `consoleblank=n` to have the console blank after `n` seconds of inactivity, for example `consoleblank=300` for 300 seconds = 5 minutes. To disable screen blanking, set `consoleblank=0`. Make sure that you add your `consoleblank` option to the single line of text already in the `cmdline.txt` file.
+
+You can also use the `raspi-config` tool to disable screen blanking. Note that the screen blanking setting in `raspi-config` also controls screen blanking when the graphical desktop is running.
```

Comment:
Perhaps this sentence would be less ambiguous if "screen blanking" was hyphenated like "screen-blanking" ?

Reply:
> Perhaps this sentence would be less ambiguous if "screen blanking" was hyphenated like "screen-blanking" ?

A hyphen would be gramatically incorrect.